### PR TITLE
Feat 28 textbook list

### PR DESCRIPTION
**Pull request by LULIU-9**
Screenshot 2023-07-08 at 1 43 53 pm
Screenshot 2023-07-08 at 1 44 03 pm
Screenshot 2023-07-08 at 1 44 16 pm
Screenshot 2023-07-08 at 1 44 24 pm

### CODE REVIEW

File: src/components/Textbook/WordDetail/index.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  encountered: number;
+  learned: number;
+  bestSeries: number;
+  handleLearnedToggle: (wordName: string) => void;
+  handleDictionaryToggle: (wordName: string) => void;
+}
+
+const WordDetail = (props: wordDetailProps) => {
+  return (
+    <div className={Styles.wordDetail}>
```
Comment (Double-Twelve) [inline src/components/Textbook/WordDetail/index.tsx:25]: You can destructure the incoming variable and use it directly
`const {name, synonyms, phonetic, definition, example, isLearnt, inDictionary, imgSrc, encountered, learned, bestSeries, handleLearnedToggle, handleDictionaryToggle} = props`
you don’t need to get the value through prop every time afterwards

File: src/components/Textbook/PageNumber.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import Button from '../Shared/Buttons';
+import Styles from './textbook.module.scss';
+const PageNumber = () => {
+  return (
+    <div className={Styles.pageNumber}>
+      <Button outline={true} color={'cyan'} label="←" size="round" />
+      <Button outline={true} color={'cyan'} label="1" size="round" />
+      <Button outline={false} color={'cyanDark'} label="2" size="round" />
+      <Button outline={true} color={'cyan'} label="3" size="round" />
+      <p className="paragrahp--p1">...</p>
```
Comment (Double-Twelve) [inline src/components/Textbook/PageNumber.tsx:11]: image
the vertical direction of this point needs to be aligned to the bottom

File: src/components/Textbook/index.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+              handleDictionaryToggle={handleDictionaryToggle}
+            />
+          ))
+        ) : (
+          <Empty />
+        )}
+      </section>
+      <section>
+        <PageNumber />
+      </section>
```
Comment (Double-Twelve) [inline src/components/Textbook/index.tsx:106]: image
if content is empty, doesn't need page number

File: src/components/Textbook/PageTitle/PageTitle.module.scss [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+            background-color: $cyanDark;
+            svg{
+              color:$white
+            }
+          } 
+        }
+        .dropdownContainer{
+          position: absolute;
+          width:max-content;
+          top:40px;
```
[the remark this answers is not in the document]
Comment (LULIU-9) [reply src/components/Textbook/PageTitle/PageTitle.module.scss]: Thanks for your feedback.  I've already discussed this with Tato, and we agreed that the current container position would improve the hover experience.

[the remark this answers is not in the document]
Comment (LULIU-9) [reply src/components/Textbook/PageTitle/PageTitle.module.scss]: > `top` should be `50px` should add `border: 1px solid $greyLight;`

Thanks for your feedback. I've already discussed this with Tato, and we agreed that the current container position would improve the hover experience.

**Review by Double-Twelve — APPROVED** [review]

**Pull request merged**